python/ray/exceptions.py

```diff
@@ -378,8 +379,12 @@ class OutOfMemoryError(RayError):
 
     # TODO: (clarng) expose the error message string here and format it with proto
     def __init__(self, message):
-        self.message = message
-
+        top_n = (
+            "\nThe top 10 memory consumers are:\n"
+            f"{memory_monitor.get_top_n_memory_usage(10)}"
```

Comment:
Is this always called on the correct node? I see the class is constructed during deserialize, which may be a different node from the source of the error IIRC.